src/shims-vue.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module '*.vue' {
+  import { DefineComponent } from 'vue'
+  const component: DefineComponent<{}, {}, any>
+  export default component
+}
```

Comment:
Is there a reason for this TypeScript definition?

Reply:
To let ts know vue syntaxes I think, see http://www.lucklnk.com/godaddy/details/aid/800872939